rumqttd/src/server/broker.rs

```diff
@@ -568,12 +568,18 @@ async fn remote<P: Protocol>(
         // No need to send a disconnect message when disconnection
         // originated internally in the router.
         Err(remote::Error::Link(e)) => {
-            error!(error=?e, "router-drop");
```

Comment:
we don't need to add client_id like this, as it is recorded by the span.
previously this was there: 
```rust
Span::current().record("client_id", &client_id);
```
but it was removed in #566 . I'm unsure of the reason, but will check with @mnpw once and get back to you!

ps:  let's keep this PR only related to `ConnectionAbort` related change, so we can unblock, plus it is always better to tackle one particular thing in PR ( makes it easier to track down in future ). So can you please remove the `client_id` related changes ( here + in changelog ) so we can merge it!

Reply:
Sure.